.aws/faucet.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-AMOUNT=100stake
+AMOUNTS=5stake
```

Comment:
like said yesterday: this will display as `0.0000005` stake. I vote for increasing all values by `10e6`

Reply:
yep good call!

Reply:
pumped all money upmillion